core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -60,22 +62,19 @@ class TransactionStateManager(brokerId: Int,
 
   this.logIdent = "[Transaction Log Manager " + brokerId + "]: "
 
-  type WriteTxnMarkers = WriteTxnMarkerArgs => Unit
+  type SendTxnMarkersCallback = (String, Int, TransactionMetadata, TransactionResult) => Unit
```

Comment:
EDIT: the reason I made it as unnamed parameters is that the `sendTxnMarkers` function is not long used as a parameter but also directly called elsewhere. And it has been reduced from 8 params to 4, so I feel this is better cost-effective?